components/Navigation.tsx

```diff
@@ -446,6 +446,11 @@ const navigation: navigationItem[] = [
         title: 'Avo Functions in monorepos',
         path: '/implementation/avo-in-monorepo',
       },
+      {
+        type: 'route',
+        title: 'Explicit `null` in the Codegen',
```

Comment:
I think we should avoid the back-ticks in sidebar items (they don't get rendered)
```suggestion
        title: 'Explicit null in the Codegen',
```